closed/OpenJ9.gmk

```diff
@@ -28,11 +28,22 @@ ifeq (,$(BUILD_ID))
   BUILD_ID := 000000
 endif
 
+OPENJ9_BRANCH := $(shell git -C $(OPENJ9_TOPDIR)    rev-parse --abbrev-ref HEAD)
+# find the closest tag name
+OPENJ9_TAG    := $(shell git -C $(OPENJ9_TOPDIR)    describe --abbrev=0)
```

Comment:
Running this locally results in:
```
$ git describe --abbrev=0
fatal: No names found, cannot describe anything.
```
which won't work with the `ifeq (,$(OPENJ9_TAG))` check below.

Reply:
Though I can't find a makefile reference link, it appears to me that the `$(shell)` function captures the result of `STDOUT` not `STDERR`, i.e., `STDOUT` is empty and `STDERR` is `fatal: No names found, cannot describe anything`.  In case there is no `tag` associated with current `git repo`, a personal build SDK with this PR was able to detect the `OPENJ9_TAG` is empty and use `OPENJ9_BRANCH-OPENJ9_SHA` instead.